### ISSUE TITLE
Version 3.0 doesn't install properly with cocoapods

### ISSUE THREAD

**Issue by bitwit** [sberrevoets/SDCAlertView#114]
I tried to install v2.5.4 without any issue but this was my log output when I tried 3.0:

```
Updating local specs repositories
Analyzing dependencies
Downloading dependencies
Installing SDCAlertView (3.0)

[!] Error installing SDCAlertView
[!] /usr/local/bin/git clone https://github.com/sberrevoets/SDCAlertView.git /var/folders/z_/y0pzsg_d4clc2kpwhtmdf1400000gn/T/d20151010-83876-be9cco --single-branch --depth 1 --branch 3.0

Cloning into '/var/folders/z_/y0pzsg_d4clc2kpwhtmdf1400000gn/T/d20151010-83876-be9cco'...
warning: Could not find remote branch 3.0 to clone.
fatal: Remote branch 3.0 not found in upstream origin
Unexpected end of command stream
```

**Comment by sberrevoets** [comment]
What does your Podfile look like? Any chance you specified the 3.0 branch in there? I deleted that last night since 3.0 is now just in master.

**Comment by bitwit** [comment]
I copy-pasted from the provided example in the README i.e. 
```
platform :ios, '8.0'
use_frameworks!

target 'myapp' do
pod 'SDCAlertView', '~> 3.0'
end
```

I also tried removing the '~> 3.0' completely, wiped my Pods/ folder and installed and it seemed that pod resolved to using `3.0` and failed as above

**Comment by sberrevoets** [comment]
Ahh, yes, this was an error in the pod spec. I have updated fixed this in 3.0.1, which I just released. Sorry for the inconvenience, please try again.
